submarine-server/server-core/pom.xml

```diff
@@ -437,6 +437,11 @@
           <artifactId>jackson-annotations</artifactId>
         </exclusion>
       </exclusions>
+    </dependency>
+	<dependency>
+      <groupId>joda-time</groupId>
+      <artifactId>joda-time</artifactId>
+      <version>2.10.8</version>
```

Comment:
- should better move the dependency to center management
- the artifact's version should define in property